_includes/navbar.html

```diff
@@ -20,12 +20,12 @@
       </div>
       <ul class="usa-button-list usa-unstyled-list">
         <li>
-          <a class="usa-button" href="#" onclick="ga('send', 'event', 'Downloaded framework', 'Clicked download button inside site');">Download</a>
+          <a class="usa-button" href="#" onclick="ga('send', 'event', 'Downloaded framework', 'Clicked download button inside site');">Download code</a>
         </li>
         <li>
           <a class="usa-button usa-button-outline" href="https://github.com/18F/usfwds/" onclick="ga('send', 'event', 'Viewed on Github', 'Clicked View on Github from inside site');"
 >
-            View on GitHub
+            View code on GitHub
```

Comment:
I'd prefer "View on GitHub." Code seems implied. If it's not adding something meaningful I'll always opt for the pithier version.